tests/test_sqlite.py

```diff
@@ -37,11 +64,52 @@ def test_sqlite_functions(tmp_path):
     # Test if the right number of spectra are returned
     assert len(spectra_list) == 2, "Expected only 2 spectra"
 
-    # Test if the correct metadata is returned for both spectra
-    spectra_json = json.load(open(path_to_json_test_file))
-    for key in spectra_list[0].metadata:
-        assert spectra_json[0][key] == spectra_list[0].metadata[key], \
-            "Expected different metadata for this spectrum"
-    for key in spectra_list[1].metadata:
-        assert spectra_json[2][key] == spectra_list[1].metadata[key], \
-            "Expected different metadata for this spectrum"
+    # Test if the correct spectra are loaded
+    pickled_file_name = os.path.join(path_to_test_files_sqlite_dir,
+                                     "first_10_spectra.pickle")
+    original_spectra = load_pickled_file(pickled_file_name)
+    assert original_spectra[0].__eq__(spectra_list[0]), \
+        "expected different spectrum"
+    assert original_spectra[2].__eq__(spectra_list[1]), \
+        "expected different spectrum"
+
+    print(tanimoto_score_dataframe)
+    print(spectra_list)
+
+
+if __name__ == "__main__":
```

Comment:
I would remove the following code example from this test file. We can later add instructive code examples to the docstrings of the respective functions instead.

Reply:
Yes, I agree, this was mostly to make it easier for you to test loading the large files.